pkg/constants/constants.go

```diff
@@ -53,23 +53,24 @@ const (
 	NoDescription = "No description provided"
 )
 
-const OctopusLogo = `            &#BGGGGGGB#&
-         &GP%%%%%%%%%%%%%G&
-       &G%%%%%%%%%%%%%%%%%%P&
-      &%%%%%%%%%%%%%%%%%%%%%%#
-      %%%%%%%%%%%%%%%%%%%%%%%%
-     #%%%%%%%%%%%%%%%%%%%%%%%%&
-     #%%%%%%%%%%%%%%%%%%%%%%%%&
-      P%%%%%%%%%%%%%%%%%%%%%%#
-      &%%%%%%%%%%%%%%%%%%%%%G
-       B%%%%%%%%%%%%%%%%%%%%#
-       P%%%%%%%%%%%%%%%%%%%%P
-     &P%%%%%%%%%%%%%%%%%%%%%%P#
-   #G%%%%P%%%%%%GP%%%%%%PP%%%%%PG&
-&G%%%PB&#P%%%%G   G%%%%P  &P%%P#&&
- &&     G%%%%#    &%%%%G    #P%%#
-       &%%%G&      G%%%B       &&
-        &#          BPB`
+const OctopusLogo = `                ####
```

Comment:
Using the logo as it was previously was causing some strange printing errors, I think because of all the `%` characters. I think it also looked a bit strange with all the different characters, so have changed it to just use `#`. I think it looks a bit nicer like this but definitely happy to discuss if there was a good reason for the characters that I'm not aware of or others disagree on the aesthetics.